planutils/__init__.py

```diff
@@ -14,19 +25,9 @@ def setup():
         else:
             return
 
-    CUR_DIR = os.path.dirname(os.path.abspath(__file__))
-
-    print("\nCreating ~/.planutils...")
-    os.mkdir(os.path.join(os.path.expanduser('~'), '.planutils'))
+    minimal_setup()
     os.mkdir(os.path.join(os.path.expanduser('~'), '.planutils', 'bin'))
```

Comment:
Any reason this wouldn't be part of the minimal setup as well?

Reply:
I put only the absolute minimum into the minimal setup that's needed for running all planutils commands. I think the "bin" folder is not needed for this.